packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -412,8 +416,14 @@ class Auth {
      *
      */
     async logout() {
-        // TODO: are we missing a call to /logout?
-        // Ticket: https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001EFF4nYAH/view
+        // invalidate refresh_token and access_token in the backend
+        const refreshTokenRegistered = this.get('refresh_token_registered')
+        const refreshTokenGuest = this.get('refresh_token_guest')
+        const refreshToken: string = refreshTokenRegistered || refreshTokenGuest
+        await helpers.logout(this.client, {
+            refreshToken: refreshToken,
+            accessToken: this.get('access_token')
+        })
```

Comment:
🤔 do we care about the result of the logout call? What do we want to do if this fails (errors are not handled) ? Is this a "send request and forget about it" situation, if so, we should remove `await`.